src/Controller/Controller.php

```diff
@@ -509,10 +508,20 @@ public function invokeAction(): ?ResponseInterface
             ]);
         }
 
-        /** @var callable $callable */
-        $callable = [$this, $request->getParam('action')];
+        return Closure::fromCallable([$this, $action]);
```

Comment:
Does this allow Reflection to introspect the parameters of the action?

Reply:
`ReflectionFunction` does allow introspecting closure args, I'll add a test for it